package.json

```diff
@@ -134,9 +134,8 @@
     "coveralls": "^3.0.1",
     "eslint": "^5.7.0",
     "eslint-plugin-prettier": "^3.0.0",
-    "firebase-admin": "^7.3.0",
+    "firebase-admin": "^7.0.0",
```

Comment:
It probably doesn't matter (as this is a dev dependency), but why downgrade?

Reply:
Just for completeness, we only check major version in functions runtime. It shouldn't matter, it was only `7.3.0` because that's what npm added.